src/Compute/Compute/help/New-AzHostGroup.md

```diff
@@ -39,6 +39,14 @@ Tags                     : {[key1, val1]}
 
 This command creates a host group in the given location and zone.
 
+### Example 2
+
+Creates a host group. (autogenerated)
+
+```powershell <!-- Aladdin Generated Example --> 
+New-AzHostGroup -Location <String> -Name <String> -PlatformFaultDomain <Int32> -ResourceGroupName <String>
+```
```

Comment:
We should omit examples where less than 50% of the specified parameters have actual values. Specifying the type for all of the parameters doesn't provide much value as the syntax can already be obtained from the cmdlet.